launch_ros/launch_ros/event_handlers/on_state_transition.py

```diff
@@ -33,8 +32,8 @@ class OnStateTransition(EventHandler):
     def __init__(
         self,
         *,
-        entities: SomeActionsType,
-        target_lifecycle_node: LifecycleNode = None,
+        entities: SomeEntitiesType,
```

Comment:
This isn't `import`ed and it doesn't exist in Humble: https://github.com/ros2/launch/blob/rolling/launch/launch/some_entities_type.py